pkg/rbd/rbd_util.go

```diff
@@ -64,22 +66,24 @@ type rbdVolume struct {
 	//   JSON tag as it is not stashed in JSON encoded config maps in v1.0.0
 	// VolName and MonValueFromSecret are retained from older plugin versions (<= 1.0.0)
 	//   for backward compatibility reasons
-	RbdImageName       string
-	VolID              string `json:"volID"`
-	Monitors           string `json:"monitors"`
-	Pool               string `json:"pool"`
-	DataPool           string
-	ImageFormat        string `json:"imageFormat"`
-	ImageFeatures      string `json:"imageFeatures"`
-	VolSize            int64  `json:"volSize"`
-	AdminID            string `json:"adminId"`
-	UserID             string `json:"userId"`
-	Mounter            string `json:"mounter"`
-	DisableInUseChecks bool   `json:"disableInUseChecks"`
-	ClusterID          string `json:"clusterId"`
-	RequestName        string
-	VolName            string `json:"volName"`
-	MonValueFromSecret string `json:"monValueFromSecret"`
+	RbdImageName         string
+	VolID                string `json:"volID"`
+	Monitors             string `json:"monitors"`
+	Pool                 string `json:"pool"`
+	DataPool             string
+	ImageFormat          string `json:"imageFormat"`
+	ImageFeatures        string `json:"imageFeatures"`
+	AdminID              string `json:"adminId"`
+	UserID               string `json:"userId"`
+	Mounter              string `json:"mounter"`
+	ClusterID            string `json:"clusterId"`
+	RequestName          string
+	VolName              string `json:"volName"`
+	MonValueFromSecret   string `json:"monValueFromSecret"`
+	PassphraseFromSecret string `json:"passphraseFromSecret"`
+	VolSize              int64  `json:"volSize"`
```

Comment:
Nit: why did you move these two fields (this line and below) around?

Reply:
this was due to a failure in travis - it mentions that the size of the structure can be reduced if rearrange the fields